lib/src/main/java/ca/barrenechea/widget/recyclerview/decoration/DoubleHeaderDecoration.java

```diff
@@ -213,29 +226,44 @@ public void onDrawOver(Canvas c, RecyclerView parent, RecyclerView.State state)
             final View child = parent.getChildAt(layoutPos);
             boolean visible = getAnimatedTop(child) > -child.getHeight()/* && child.getTop() < parent.getHeight()*/;
             final int adapterPos = parent.getChildAdapterPosition(child);
-            if (visible && adapterPos != RecyclerView.NO_POSITION && (!headerDrawn || hasSubHeader(adapterPos))) {
+            if (visible && adapterPos != RecyclerView.NO_POSITION) {
                 int left, top;
-                final View header = getHeader(parent, adapterPos).itemView;
-                final View subHeader = getSubHeader(parent, adapterPos).itemView;
-
-                c.save();
-                left = child.getLeft();
-                top = getSubHeaderTop(parent, child, header, subHeader, adapterPos, layoutPos);
-                c.translate(left, top);
-                subHeader.setTranslationX(left);
-                subHeader.setTranslationY(top);
-                subHeader.draw(c);
-                c.restore();
+                View header = null;
+
+                if (mAdapter.getHeaderId(adapterPos) != NO_HEADER_ID) {
```

Comment:
Actually no, hasHeader/hasSubheader returns false in other cases than this check, this make the header have scroll problem since the calculations might be wrong